src/main/scala/net/jcazevedo/moultingyaml/YamlPrinter.scala

```diff
@@ -0,0 +1,67 @@
+package net.jcazevedo.moultingyaml
+
+import org.yaml.snakeyaml.{DumperOptions, Yaml}
+
+/**
+  * @author Simone D'Avico (simonedavico@gmail.com)
```

Comment:
Scaladoc, akin to Javadoc, should precede the items it documents without any separating newline. In this case it doesn't refer to any entity, so it's not very relevant.